system-int-test/test/cmr/system_int_test/ingest/bulk_update/bulk_update_test.clj

```diff
@@ -181,50 +181,61 @@
                           :update-value {:Category "EARTH SCIENCE"
                                          :Topic "HUMAN DIMENSIONS"
                                          :Term "ENVIRONMENTAL IMPACTS"
-                                         :VariableLevel1 "HEAVY METALS CONCENTRATION"}}]
-       (side/eval-form `(ingest-config/set-bulk-update-enabled! false))
-       ;; Kick off bulk update
-       (let [response (ingest/bulk-update-collections "PROV1" bulk-update-body)]
-         (is (= 400 (:status response)))
-         (is (= ["Bulk update is disabled."] (:errors response))))
-       ;; Wait for queueing/indexing to catch up
-       (index/wait-until-indexed)
-       (let [collection-response (ingest/bulk-update-task-status "PROV1" 1)]
-         (is (= 404 (:status collection-response)))
-         (is (= ["Bulk update task with task id [1] could not be found."]
-                (:errors collection-response))))
+                                         :VariableLevel1 "HEAVY METALS CONCENTRATION"}}
+        ;; CMR-4570 tests that no duplicate science keywords are created.
+        duplicate-body {:concept-ids concept-ids
+                        :name "TEST NAME"
+                        :update-type "ADD_TO_EXISTING"
+                        :update-field "SCIENCE_KEYWORDS"
+                        :update-value {:Category "EARTH SCIENCE"
+                                       :Term "MARINE SEDIMENTS"
+                                       :Topic "OCEANS"}}]
+    (side/eval-form `(ingest-config/set-bulk-update-enabled! false))
+    ;; Kick off bulk update
+    (let [response (ingest/bulk-update-collections "PROV1" bulk-update-body)]
+      (is (= 400 (:status response)))
+      (is (= ["Bulk update is disabled."] (:errors response))))
+    ;; Wait for queueing/indexing to catch up
+    (index/wait-until-indexed)
+    (let [collection-response (ingest/bulk-update-task-status "PROV1" 1)]
+      (is (= 404 (:status collection-response)))
+      (is (= ["Bulk update task with task id [1] could not be found."]
+             (:errors collection-response))))
 
-       (side/eval-form `(ingest-config/set-bulk-update-enabled! true))
-       ;; Kick off bulk update
-       (let [response (ingest/bulk-update-collections "PROV1" bulk-update-body)]
-         (is (= 200 (:status response)))
-         ;; Wait for queueing/indexing to catch up
-         (index/wait-until-indexed)
-         (let [collection-response (ingest/bulk-update-task-status "PROV1" (:task-id response))]
-           (is (= "COMPLETE" (:task-status collection-response))))
+    (side/eval-form `(ingest-config/set-bulk-update-enabled! true))
+    ;; Kick off bulk update
+    (let [response (ingest/bulk-update-collections "PROV1" bulk-update-body)
+          ;; Initiate bulk update that shouldn't add anything, including duplicates.
+          _ (index/wait-until-indexed)
```

Comment:
These two non-assignments don't need to be (and shouldn't be) in the `let` block; they can accomplish their tasks just fine before the `(is ...)` call.